runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/SkillingAnimations.java

```diff
@@ -0,0 +1,95 @@
+package net.runelite.client.plugins.idlenotifier;
+
+import java.util.HashMap;
+import java.util.Map;
+import static net.runelite.api.AnimationID.*;
+
+enum SkillingAnimations
+{
+	WOODCUTTING("Woodcutting",
+		WOODCUTTING_BRONZE, WOODCUTTING_IRON, WOODCUTTING_STEEL,
+		WOODCUTTING_BLACK, WOODCUTTING_MITHRIL, WOODCUTTING_ADAMANT,
+		WOODCUTTING_RUNE, WOODCUTTING_DRAGON, WOODCUTTING_INFERNAL,
+		WOODCUTTING_3A_AXE
+	),
+	COOKING("Cooking",
+		COOKING_FIRE, COOKING_RANGE
+	),
+	CRAFTING("Crafting",
+		GEM_CUTTING_OPAL, GEM_CUTTING_JADE, GEM_CUTTING_REDTOPAZ,
+		GEM_CUTTING_SAPPHIRE, GEM_CUTTING_EMERALD, GEM_CUTTING_RUBY,
+		GEM_CUTTING_DIAMOND, CRAFTING_GLASSBLOWING, CRAFTING_SPINNING
+	),
+	FLETCHING("Fletching",
+		FLETCHING_BOW_CUTTING, FLETCHING_STRING_NORMAL_SHORTBOW, FLETCHING_STRING_OAK_SHORTBOW,
+		FLETCHING_STRING_WILLOW_SHORTBOW, FLETCHING_STRING_MAPLE_SHORTBOW, FLETCHING_STRING_YEW_SHORTBOW,
+		FLETCHING_STRING_MAGIC_SHORTBOW, FLETCHING_STRING_NORMAL_LONGBOW, FLETCHING_STRING_OAK_LONGBOW,
+		FLETCHING_STRING_WILLOW_LONGBOW, FLETCHING_STRING_MAPLE_LONGBOW, FLETCHING_STRING_YEW_LONGBOW,
+		FLETCHING_STRING_MAGIC_LONGBOW
+	),
+	SMITHING("Smithing",
+		SMITHING_ANVIL, SMITHING_SMELTING, SMITHING_CANNONBALL
+	),
+	FISHING("Fishing",
+		FISHING_NET, FISHING_BIG_NET, FISHING_HARPOON,
+		FISHING_BARBTAIL_HARPOON, FISHING_DRAGON_HARPOON, FISHING_CAGE,
+		FISHING_POLE_CAST, FISHING_INFERNAL_HARPOON, FISHING_OILY_ROD,
+		FISHING_KARAMBWAN, FISHING_CRUSHING_INFERNAL_EELS, FISHING_BAREHAND
+	),
+	MINING("Mining",
+		MINING_BRONZE_PICKAXE, MINING_IRON_PICKAXE, MINING_STEEL_PICKAXE,
+		MINING_BLACK_PICKAXE, MINING_MITHRIL_PICKAXE, MINING_ADAMANT_PICKAXE,
+		MINING_RUNE_PICKAXE, MINING_DRAGON_PICKAXE, MINING_DRAGON_PICKAXE_ORN,
+		MINING_INFERNAL_PICKAXE, MINING_3A_PICKAXE, MINING_MOTHERLODE_BRONZE,
+		MINING_MOTHERLODE_IRON, MINING_MOTHERLODE_STEEL, MINING_MOTHERLODE_BLACK,
+		MINING_MOTHERLODE_MITHRIL, MINING_MOTHERLODE_ADAMANT, MINING_MOTHERLODE_RUNE,
+		MINING_MOTHERLODE_DRAGON, MINING_MOTHERLODE_DRAGON_ORN, MINING_MOTHERLODE_INFERNAL,
+		MINING_MOTHERLODE_3A
+	),
+	HERBLORE("Herblore",
+		HERBLORE_POTIONMAKING, HERBLORE_MAKE_TAR
+	),
+	PRAYER("Prayer",
+		USING_GILDED_ALTAR
+	),
+	OTHER("Other",
```

Comment:
Just name this magic I guess?

Reply:
It's a very minor part of magic, and can have other random activities added to it in the future so we don't get menu entry swapper 2.0

Reply:
Well, category "Other" is basically "dump everything that do not fits here" and that is bad thing to do.